tasks/src/reinforcement-learning/about.md

```diff
@@ -1,6 +1,14 @@
 ## Use Cases
 
-You can contribute this area with common use cases of the task! 
+### Gaming
+
+Reinforcement learning is known for its application to video games. Since the games provide a safe environment to the agent to be trained in the sense that it is perfectly defined and controllable. This makes them perfect candidates for experimentation and will help a lot to learn about the capabilities and limitations of various RL algorithms. 
+
+Many example videos can be found on the Internet where the RL agent starts with a really bad gaming strategy due to random initialization but gets better and better with each episode of the training. This [paper](https://arxiv.org/abs/1912.10944) particularly investigates the performance of RL on popular games such as Minecraft or Dota2. Usually super-human level can be reached although there are still some challenges mostly related to efficiency to construct the gaming policy of the RL agent.
+
+### Healthcare
+
+RL techniques can be used to recommend optimal treatments to the patients based on certain readings of the patients like vital signs or laboratory measurements. The usage of RL is advantageous because other automated systems in healthcare are based on models that rely on human knowledge whereas RL constructs the necessary models using past experiences. The optimal actions in this case can be dosage of a medication, optimal settings of a medical machine etc.

```

Comment:
We try to avoid giving examples of use of ML in life-critical use cases.

Reply:
Ok, this is understandable. For reasons I have mentioned in previous comment I cannot make a pull so I will appreciate if you can remove it.

May I ask if we consider the self-driving car as a life-critical case? I believe it may add interesting things to this title.